<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventoryinquiry</class>
 <widget class="QWidget" name="Inventoryinquiry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>492</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;

font: 75 14pt &quot;Aparajita&quot;;</string>
   </property>
   <property name="title">
    <string>Material Inquiry</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>611</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>611</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Inquiry Model:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="GoodIDradioButton">
         <property name="text">
          <string>Goods ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="GoodsNameradioButton">
         <property name="text">
          <string>Goods Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="searchpushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgb(153, 149, 137); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: black; }
</string>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backpushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgb(153, 149, 137); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: black; }
</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
